src/main/java/com/bp3x/raidbot/commands/RemindCommand.java

```diff
@@ -0,0 +1,65 @@
+package com.bp3x.raidbot.commands;
+
+import com.bp3x.raidbot.commands.util.Event;
+import com.bp3x.raidbot.commands.util.LFGEmbedBuilder;
+import com.bp3x.raidbot.util.RaidBotEmbedBuilder;
+import com.bp3x.raidbot.util.RaidBotRuntimeException;
+import com.jagrosh.jdautilities.command.Command;
+import com.jagrosh.jdautilities.command.CommandEvent;
+import net.dv8tion.jda.api.entities.Member;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Reminder Command to ping accepted players.
+ */
+public class RemindCommand extends Command {
+    private final Logger log = LoggerFactory.getLogger(RemindCommand.class);
+
+
+
+    public RemindCommand() {
+        this.name = "remind";
+        this.help = "Use to remind accepted players";
+        this.arguments = "<eventID> <playerCategory>";
+        this.guildOnly = false;
+        this.ownerCommand = false;
+        this.category = new Category("General");
+
+    }
+
+    @Override
+    protected void execute(CommandEvent event) {
+        log.info("Reminder command by: " + event.getAuthor().getName() + "#" + event.getAuthor().getDiscriminator());
+        log.info("Reminding for " + event.getArgs());
+
+        String[] args = event.getArgs().split("\\s+");
+        if (args.length > 0) {
+            try {
+                // Retrieve event ID data and then add the names of the users in the accepted list
+                Event remindEvent = Event.getEventById(args[0]);
+                if (remindEvent != null) {
+                    if (Event.eventExists(remindEvent.getShortName())) {
```

Comment:
This doesn't look necessary. You're checking to see whether the event's shortName is valid, but it already is because it's already been created

Reply:
My reasoning here was checking for event existence and then whether it's actually an event that is currently valid. However, upon further reflection there is no way that an event can exist while not being valid so I'll fix this.